docs/conf.py

```diff
@@ -152,18 +152,16 @@
 #
 html_theme = "sphinx_book_theme"
 html_logo = "_static/autoplex_logo.png"
+html_favicon = "_static/autoplex_favicon.png"
 html_theme_options = {
     "repository_provider": "github",
     "repository_url": "https://github.com/JaGeo/autoplex",
     "use_repository_button": True,
     "use_issues_button": True,
 }
 
-
 # html_static_path = ["_static"]
 
-html_css_files = ['custom.css']

```

Comment:
I noticed you have move this to line above. But there is no `github.css` file in our `docs/_static` directory. Is that you wanted to add at some point and it got left out of commits somehow?

Reply:
It was like this all the time, I simply removed the redundant line. Maybe I can remove github.css then?